x-pack/legacy/plugins/maps/public/angular/map_controller.js

```diff
@@ -66,6 +67,32 @@ const REACT_ANCHOR_DOM_ELEMENT_ID = 'react-maps-root';
 
 const app = uiModules.get(MAP_APP_PATH, []);
 
+function getInitialLayersFromUrlParam() {
+  const locationSplit = window.location.href.split('?');
```

Comment:
Also here $location could be used

Reply:
See comment above. I do not want to proliferate angular dependencies.